### ISSUE TITLE
Last bits of PNI a11y cleanup

### ISSUE THREAD

**Issue by Pomax** [mozilla/foundation.mozilla.org#4722]
Last fixes to PNI a1yy after every a11y PR landed. Closes #4006:

## homepage

- [x] Text for `The 😮 below shows how creepy users find these products. ` should be `gray-60`
- [x] Footer text "Protect the internet as a global public resource" should be `<p`> with h5-heading CSS, no an actual `<h5>`
- [x] Footer text "More about us" should be a `<p>` as above

## Category page

- [x] company + name should be `<p>`, not `<h3>`

## About page

- [x] side nav should use `gray-60`
- [x] inner `<main>` should be `<div>`
